tests/test.py

```diff
@@ -53,7 +68,7 @@ def _check_response(r):
     assert len(response['seg_map']) == response['image_size'][1]
 
     assert response['seg_map'][0][0] == 0  # there are no objects in the top left corner
-    assert response['seg_map'][128][128] == 15  # there is a person here
+    assert response['seg_map'][129][129] == 15  # there is a person here
```

Comment:
As discussed offline can you check a pixel in the center of the person?

```suggestion
    assert response['seg_map'][130][170] == 15  # there is a person here
```